src/main/java/donnafin/logic/parser/AddressBookParser.java

```diff
@@ -49,6 +54,21 @@ public Command parseCommand(String commandWord, String arguments) throws ParseEx
         case HelpCommand.COMMAND_WORD:
             return !arguments.equals("") ? throwsInvalidInputMsg() : new HelpCommand();
 
+        case HomeCommand.COMMAND_WORD:
+            return !arguments.equals("") ? throwsInvalidInputMsg() : new HomeCommand();
```

Comment:
Do u want this to be fallthrough as well and say that it is not possible. No point returning new home command in addressbookparser. Just brings it back to the same thing

Reply:
I think it will be quite odd for home command to not be available here. Users can use it and yes, it does bring it back to the same thing but it feels more complete. Using home command would show the user they are already in home window. Since the displayed message is "Currently viewing home window.", the behaviour is still correct. What do you think?

Reply:
I think "Currently viewing home window" is the right response. Its lucky its not "Switching to viewing home" or smth like that. Thanks for exploring that avenue tho, tee chin.